integrations/ipfy.go

```diff
@@ -0,0 +1,66 @@
+package integrations
+
+import (
+	"github.com/buger/jsonparser"
+	"io/ioutil"
+	"net"
+	"net/http"
+)
+
+const IpFyExtIPEndpoint = "https://api.ipify.org?format=json"
```

Comment:
Codacy found an issue: [exported const IpFyExtIPEndpoint should have comment or be unexported](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)